src/main/java/frc/robot/RobotContainer.java

```diff
@@ -70,6 +74,46 @@ public class RobotContainer {
   private final Timer coastModeTimer = new Timer();
   private final StringLogEntry phaseLogger = new StringLogEntry(LOG, "/Robot/Phase");
 
+  public enum RobotSelector {
+    PracticeRobot2025(AprilTag.VISION_PARAMS),
+    CompetitionRobot2025(AprilTag.VISION_PARAMS);
+
+    private final VisionParameters visionParams;
+
+    private RobotSelector(VisionParameters visionParams) {
+      this.visionParams = visionParams;
+    }
+
+    public VisionParameters visionParameters() {
+      return visionParams;
+    }
+  }
+
+  @RobotPreferencesValue
+  public static RobotPreferences.EnumValue<RobotSelector> PARAMETERS =
+      new RobotPreferences.EnumValue<RobotSelector>(
```

Comment:
Should we just `import com.nrg948.preferences.RobotPreferencesValue.EnumValue` to reduce line length?